src/ApiCompat/Rules/Compat/CannotMakeMoreVisible.cs

```diff
@@ -23,7 +23,7 @@ public override DifferenceType Diff(IDifferences differences, ITypeDefinitionMem
             // If implementation is protected then contract must be protected as well. 
             if (impl.Visibility == TypeMemberVisibility.Family)
```

Comment:
I think you also want to update the condition to add ` || impl.Visibility == TypeMemberVisibility.FamilyOrAssembly`.